src/components/header/navigation/Navigation.module.css

```diff
@@ -0,0 +1,4 @@
+.header_bg {
```

Comment:
Mantine supports `css-in-js`, can you explain the reason why you used plain CSS?